test/test-rotateElement.js

```diff
@@ -0,0 +1,390 @@
+        let elementTetromino = tetris.getCoordinates(`
+            -----
+            -----
+            -----
+            -----
+            ---X-
+        `)
+        let pivot = tetris.getCoordinates(`
+            -----
+            -----
+            --X--
+            -----
+            -----
+        `)
+
+        let stepElementTetromino = tetris.rotateElement(elementTetromino[0], pivot[0])
+        let expectedResult = tetris.getCoordinates(`
+            -----
+            ----X
+            -----
+            -----
+            -----
+        `)
+    })
+})
+describe("Rotation where starts to the bottom, pivot has different x and y coordinates", function () {
+    it("element move the left field 6x5 ", function () {
+        let elementTetromino = tetris.getCoordinates(`
+            -----
+            -----
+            -----
+            -----
+            -----
+            -X---
+        `)
+        let pivot = tetris.getCoordinates(`
+            -----
+            -----
+            -X---
+            -----
+            -----
+            -----
+        `)
+        let stepElementTetromino = tetris.rotateElement(elementTetromino[0], pivot[0])
+        let expectedResult = tetris.getCoordinates(`
+            -----
+            -----
+            ----X
+            -----
+            -----
+            -----
+        `)
+    })
+
+    it("element move the top field 6x5 ", function () {
+        let elementTetromino = tetris.getCoordinates(`
+            -----
+            -----
+            ----X
+            -----
+            -----
+            -----
+        `)
+        let pivot = tetris.getCoordinates(`
+            -----
+            -----
+            -X---
+            -----
+            -----
+            -----
+        `)
+        let stepElementTetromino = tetris.rotateElement(elementTetromino[0], pivot[0])
+        let expectedResult = [{ x: 1, y: -1 }]
+    })
+
+    it("element move the top field 6x5 ", function () {
+        let elementTetromino = [{ x: 1, y: -1 }]
+        let pivot = tetris.getCoordinates(`
+            -----
+            -----
+            -X---
+            -----
+            -----
+            -----
+        `)
+        let stepElementTetromino = tetris.rotateElement(elementTetromino[0], pivot[0])
+        let expectedResult = [{ x: 1, y: -1 }]
+    })
+
+    it("element move the top field 6x5 ", function () {
+        let elementTetromino = [{ x: 1, y: -1 }]
+        let pivot = tetris.getCoordinates(`
+            -----
+            -----
+            -X---
+            -----
+            -----
+            -----
+        `)
+        let stepElementTetromino = tetris.rotateElement(elementTetromino[0], pivot[0])
+        let expectedResult = [{ x: -2, y: 2 }]
+    })
+})
+describe("Rotation where pivot coordinates coincides with element coordinates", function () {
+    it("element not move field 4x4 ", function () {
+        let elementTetromino = tetris.getCoordinates(`
+           ----
```

Comment:
please indent 1 space to the right - so it will be the same as pivot and expectedResult

Reply:
the pivot should space to the right?

Reply:
No, move each line of graphical representation of elementTetromino 1 space to the right

Reply:
Please resolve this as well